changelog/breaking.md

```diff
@@ -0,0 +1 @@
+Changed the on_resyn generator to expect a list instead of a single item
```

Comment:
```suggestion
Changed the `on_resync` generator to expect a list instead of a single item
```

Reply:
Shouldn't this say: Updated the `on_resync` generator to support both a single item and a list of items